packages/widget-passage-citation-toc/src/PassageCitationTOCWidget.vue

```diff
@@ -0,0 +1,112 @@
+<template v-if="textParts">
+  <div class="passage-citation-toc-widget">
+    <div class="grid-cell-square" v-for="part in textParts" :key="part.urn">
+      <ReaderLink :class="{ 'active-textpart': part.selected }" :urn="part.urn">
+        {{ part.lcp }}
+      </ReaderLink>
+    </div>
+  </div>
+</template>
+
+<script>
+  import gql from 'graphql-tag';
+  import { ReaderLink } from '@scaife-viewer/common';
+  import { MODULE_NS } from '@scaife-viewer/store';
+
+  const displayNameCallback = $root => {
```

Comment:
I think this would be slightly better passing in `getters` or even a `label` that is/can-be a callable and `config` instead of `$root`.